peloader/winapi/Paths.c

```diff
@@ -42,6 +42,32 @@ UINT WINAPI GetDriveTypeW(PWCHAR lpRootPathName)
     return DRIVE_FIXED;
 }
 
+DWORD WINAPI GetLongPathNameA(LPCSTR lpszShortPath,
+                              LPSTR lpszLongPath,
+                              DWORD cchBuffer)
+{
+    // For now we just return the 8.3 format path as the long path
+    if (cchBuffer > strlen(lpszShortPath)) {
+        memcpy(lpszLongPath, lpszShortPath, sizeof(lpszShortPath));
+    }
+
+    return strlen(lpszShortPath);
+}
+
+DWORD WINAPI GetLongPathNameW(LPCWSTR lpszShortPath,
+                              LPWSTR lpszLongPath,
+                              DWORD cchBuffer)
+{
+    // For now we just return the 8.3 format path as the long path
+    if (cchBuffer > strlen(lpszShortPath)) {
```

Comment:
I think this isn't right, because `strlen()` won't work on a LPWSTR, I think you can use this:

```c
if (cchBuffer > CountWideChars(lpszShortPath)) ...
```